ckanext/datitrentinoit/model/mapping.py

```diff
@@ -153,6 +156,11 @@ def dateformat(d):
         'Measurement unit':  metadata.get_um(),
     }
 
+    extras['temporal_coverage'] = [{'temporal_start': dateformat(created)}]
```

Comment:
We don't want the temporal_start if "AnnoInizio" is not defined.
You are taking here `created`, which is *always* defined (it is created using `start_date` which has a fallback value in its creation.